src/ExtendedSelenium2Library/__init__.py

```diff
@@ -558,6 +560,14 @@ def _select_checkbox_or_radio_button(self, element):
             element.click()
             self._wait_until_page_ready()
 
+    def register_page_ready_keyword(self, keyword_name):
```

Comment:
You might want to rename it to register_page_ready_strategy, to be consistent with remove_page_ready_strategy.